pkg/notary/image_signer_test.go

```diff
@@ -17,6 +19,7 @@ import (
 	"github.com/theupdateframework/notary/storage"
 	"github.com/theupdateframework/notary/tuf/data"
 	"github.com/theupdateframework/notary/tuf/signed"
+	"github.com/tj/assert"
```

Comment:
We use https://github.com/stretchr/testify instead of tj/assert.

Reply:
Oops, the VScode auto imported the wrong thing. That explains why go.mod started to act up for the github action for running the test coverage. Will fix this, thanks for catching it :)